libraries/alicloud_apsaradb_rds_instance.rb

```diff
@@ -53,7 +53,7 @@ def initialize(opts = {})
 
     opts[:vpc_id] = @vpc_id
     vpc_info = fetch_vpc_info(opts)
-    @in_default_vpc = vpc_info.nil? ? false : vpc_info["IsDefault"]
+    @in_default_vpc = vpc_info["IsDefault"]
```

Comment:
this means, `@in_default_vpc ` will be `nil`  and not `false` if `IsDefault` key is not present is this expected?

Reply:
Yes that's the intention SAP were after here - the current implementation can return "false" for two separate reasons and does not indicate which, the one they care about here is the IsDefault key.